### DISCUSSION TITLE
GET node_modules/vite-plugin-ssr/dist/esm/client/entry.dff76076.js net::ERR_ABORTED 404 (Not Found)

### DISCUSSION THREAD

**Discussion by brillout** [vikejs/vike#686] [Discussion] [answered]
Moved from Discord:

> Hi everyone .  I noticed this error in console when running the default Vue boilerplate. Is this something I can fix easily? > Thanks.
> ```
> GET http://localhost:3000/assets/nodemodules/.pnpm/vite-plugin-ssr@0.4.91vite@4.1.4/
> nodemodules/vite-plugin-ssr/dist/esm/client/entry.dff76076.js net::ERR_ABORTED 404 (Not Found)
> ```

**Comment by brillout** [comment]
(FYI I it doesn't seem related to VPS. I'd dig around your server.)

**Comment by johnnypea** [comment]
I would say it uses the server from default Vue boilerplate https://github.com/brillout/vite-plugin-ssr/tree/main/boilerplates/boilerplate-vue-ts

**Reply by brillout** [reply]
I'd check your browser (extensions) then. No one has reported that issue yet, so I suspect some middleman on your machine doing something unexpected.

**Reply by brillout** [reply]
Or pin to an older Vite version.

**Reply by brillout** [reply]
Report back if you find the problem.

**Comment by johnnypea** [comment]
Additionally I found out when I rename any page to be only client side rendered `*.page.client.vue` its output is blank.

Do you think this can be Node (v19.7.0) related?  Or `pnpm` (v7.29.1)? I tried to have all other packages at most recent versions as well, but the result was the same.

**Reply by brillout** [reply]
> Additionally I found out when I rename any page to be only client side rendered `*.page.client.vue` its output is blank.

Seems like a user land issue. Double check your renderer.

> Do you think this can be Node (v19.7.0) related? Or `pnpm` (v7.29.1)? I tried to have all other packages at most recent versions as well, but the result was the same.

Maybe, but I'm more inclined to think it's something else about your setup.

**Reply by johnnypea** [reply]
> Seems like a user land issue. Double check your renderer.

But this is just the boilerplate project I just changed the file name. :(

**Reply by johnnypea** [reply]
I forgot to mention this works on "dev" environment.

**Reply by brillout** [reply]
https://vite-plugin-ssr.com/render-modes

**Reply by johnnypea** [reply]
So `boilerplate-vue-ts` is not ready for client only (browser) rendering? I think I get it now. I'm sorry but I was really confused by the examples.

**Comment by brillout** [comment, marked as answer]
Actually, it's a bug that slipped through the CI and that got introduced in `0.4.91`.

Bug report: https://github.com/brillout/vite-plugin-ssr/issues/696.
